_config.yml

```diff
@@ -26,6 +26,7 @@ domain: "https://usds.gov"
 
 timezone: "America/New_York"
 social_image: social-card--default.png
+social_image_alt_text: "Blue graphic reads 'Building a better government for the people' with USDS logo: navy blue shield with three stars, two stripes and yellow wings"
```

Comment:
Suggested text edit:

"Graphic with bold text that reads 'Building a better government for the people.' Text is displayed next to the USDS logo, a navy blue shield with three stars, two stripes and yellow wings."

Reply:
We should use githubs's Suggested Edits feature to make this easier?